doc/content/msr/msr_generic/seismic_analysis/npp_building/npp_building_pt1.md

```diff
@@ -1,5 +1,7 @@
 # Seismic analysis of a base-isolated nuclear power plant building
 
+This model was adopted from the list of examples on the [MASTODON](https://mooseframework.inl.gov/mastodon/examples/index.html) website.
```

Comment:
add a link to the inputs on the virtual test bed
somewhere like https://github.com/idaholab/virtual_test_bed/tree/devel/msr/msr_generic/seismic_analysis